addons/project/static/tests/project_subtask_kanban_list_tests.js

```diff
@@ -82,4 +93,23 @@ QUnit.module('Subtask Kanban List tests', {
 
         assert.containsNone(target, '.subtask_list', "If the drawdown button is clicked again, the subtasks list should be hidden again");
     });
+
+    QUnit.test("focus new subtask's name", async function (assert) {
+        assert.expect(1);
+
+        const views = this.views;
```

Comment:
`assert.expect` is not needed here. 🙂 
```suggestion
        const views = this.views;
```